corehq/util/view_utils.py

```diff
@@ -74,7 +74,11 @@ def inner(request, *args, **kwargs):
         except BadRequest as e:
             return _get_json_exception_response(400, request, e)
         except Exception as e:
-            notify_exception(request, b'JSON exception response: {}'.format(smart_bytes(e)))
+            if six.PY3:
+                message = 'JSON exception response: {}'.format(smart_text(e)).encode('utf-8')
```

Comment:
Made updates https://github.com/dimagi/commcare-hq/pull/21676/commits/68902542c366e6e7bcbc12f42cc3410e57a3bd5a and https://github.com/dimagi/commcare-hq/pull/21676/commits/ab59a436a30f1a9ef6d0efb4d4d1a06d575dd9dd

Reply:
This does mean that exceptions in python 3 with bytestring arguments will be logged with a ```b``` (see changes to tests), but I think this is a good thing because in py3 we should be aware of unintended uses of bytes and trying to replace them with unicode.